src/GHActServer.ts

```diff
@@ -22,6 +27,11 @@ type webhookPayload = {
     name: string;
     email: string;
   };
+  commits: {
+    added: string[];
+    removed: string[];
+    modified: string[];
+  }[];
```

Comment:
no, `changeSummary` also has properties `from`, `till`. Also I would’t want to use it here as the structure of `webhookPayload` is given externally and this would then essentially limit `changeSummary` from changing.